segmentstore/server/src/main/java/io/pravega/segmentstore/server/SegmentStoreMetrics.java

```diff
@@ -123,7 +141,7 @@ public void close() {
             this.storageActiveThreads.close();
         }
 
-        private void report() {
+        public void report() {
```

Comment:
Same here: can you keep this one `private` and use `MetricRegistryUtils` in the test to look for the JVM metric published for do the assertions?